google/cloud/vision/client.py

```diff
@@ -98,12 +98,22 @@ def annotate(self, image, features):
         :rtype: dict
         :returns: List of annotations.
         """
-        img = Image(image, self)
-        request = VisionRequest(img, features)
+        request = VisionRequest(image, features)
 
         data = {'requests': [request.as_dict()]}
         response = self.connection.api_request(method='POST',
                                                path='/images:annotate',
                                                data=data)
 
         return response['responses'][0]
+
+    def image(self, image_source):
+        """Get instance of Image using current client.
+
+        :type image_source: str or bytes
+        :param image_source: Byte stream of an image or a GCS URI.
+
+        :rtype: :class:`google.cloud.vision.image.Image`
+        :returns: Image instance with the current client attached.
+        """
+        return Image(client=self, image_source=image_source)
```

Comment:
I would be fine with just having separate parameters for the two usecases:

```python
image1 = client.image(content=BYTES)
image2 = client.image(source_uri='gs://my-bucket/image.jpg')
```

And likewise for `Image.__init__`.